test/functional/ui/cmdline_spec.lua

```diff
@@ -1713,20 +1713,4 @@ describe('cmdheight=0', function()
       {2:[No Name]                }|
     ]])
   end)
-
-  it('cannot be resized at all with external messages', function()
-    clear()
-    screen = new_screen({ rgb = true, ext_messages = true })
-    command('set laststatus=2 mouse=a')
-    command('resize -1')
-    screen:expect([[
-      ^                         |
-      {1:~                        }|*3
-      {3:[No Name]                }|
-    ]])
-    api.nvim_input_mouse('left', 'press', '', 0, 6, 10)
-    poke_eventloop()
-    api.nvim_input_mouse('left', 'drag', '', 0, 5, 10)
-    screen:expect_unchanged()
```

Comment:
better to update the test and its title.

Reply:
Sure, so do we follow through with allowing 'cmdheight' > 0 with ext_messages?

Reply:
yes, it should be allowed as it is required for The Plan, but it should still default to zero with an external ext_messages UI (otherwise it is just wasted space).

Reply:
I mean The Plan is still up in the air. My feeling now is that 100% mimicking the current message grid implementation to pass tests was a nice idea, but might not be worth the effort. So if we can agree on a different representation that does not use the 'cmdheight' area it might not be necessary for The Plan.

That doesn't change the fact that it can still be a valid use case, so I think we can follow through with this PR regardless.

> but it should still default to zero with an external ext_messages UI

Hmm yeah that makes sense since it's the cmdheight area is not in use unless the event handler uses it, so it should be responsible for making it non-zero. I just thought implicitly changing a (now valid) default option value was a little weird to begin with, but that is already documented. I can update the text.